resources/tests/org.eclipse.core.tests.resources/src/org/eclipse/core/tests/filesystem/FileStoreTest.java

```diff
@@ -124,35 +113,41 @@ public void testCopyAcrossVolumes() throws Throwable {
 		IFileStore[] tempDirectories = getFileStoresOnTwoVolumes();
 
 		/* test if we are in the adequate environment */
-		if (tempDirectories == null || tempDirectories.length < 2 || tempDirectories[0] == null
-				|| tempDirectories[1] == null) {
-			return;
-		}
+		Assume.assumeFalse("Can't test without at least 2 temp folders in different volumes", //
```

Comment:
i don't want a failed test just because i don't have two volumes. just skipping test under this conditions was better

Reply:
I reverted my changes, the test will now be skipped.